commit eb87f65f78bb4fc6dc532e6d7082d90ba0890cb8
Author: Jason Gaston <jason.d.gaston@intel.com>
Date:   Mon Jan 28 17:20:36 2008 -0800

    PCI: pci_ids: patch for Intel ICH10 DeviceID's
    
    This patch adds the Intel ICH10 LPC and SMBus Controller DeviceID's.
    
    Signed-off-by:  Jason Gaston <jason.d.gaston@intel.com>
    Cc: Jean Delvare <khali@linux-fr.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

Adjusted to apply to Debian's 2.6.24 by dann frazier <dannf@hp.com>

diff -urpN linux-source-2.6.24.orig/include/linux/pci_ids.h linux-source-2.6.24/include/linux/pci_ids.h
--- linux-source-2.6.24.orig/include/linux/pci_ids.h	2008-01-24 15:58:37.000000000 -0700
+++ linux-source-2.6.24/include/linux/pci_ids.h	2008-05-01 17:46:51.000000000 -0600
@@ -2339,6 +2339,12 @@
 #define PCI_DEVICE_ID_INTEL_MCH_PC1	0x359a
 #define PCI_DEVICE_ID_INTEL_E7525_MCH	0x359e
 #define PCI_DEVICE_ID_INTEL_IOAT_CNB	0x360b
+#define PCI_DEVICE_ID_INTEL_ICH10_0	0x3a14
+#define PCI_DEVICE_ID_INTEL_ICH10_1	0x3a16
+#define PCI_DEVICE_ID_INTEL_ICH10_2	0x3a18
+#define PCI_DEVICE_ID_INTEL_ICH10_3	0x3a1a
+#define PCI_DEVICE_ID_INTEL_ICH10_4	0x3a30
+#define PCI_DEVICE_ID_INTEL_ICH10_5	0x3a60
 #define PCI_DEVICE_ID_INTEL_IOAT_SNB	0x402f
 #define PCI_DEVICE_ID_INTEL_IOAT_SCNB	0x65ff
 #define PCI_DEVICE_ID_INTEL_TOLAPAI_0	0x5031
